### PR TITLE
[docs] Multi-column indexes

### LINKED ISSUE #13453: [Docs] [Explore] Document multi-column indexes
### Description

Document multi-column(composite) indexes under the Indexes and Constraints section

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #13453

Deploy preview: https://deploy-preview-18706--infallible-bardeen-164bc9.netlify.app/preview/explore/indexes-constraints/multi-column-ysql/

### CODE REVIEW

File: docs/content/preview/explore/indexes-constraints/multi-column-ycql.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+image: /images/section_icons/secure/create-roles.png
+menu:
+  preview:
+    identifier: multi-column-index-ycql
+    parent: explore-indexes-constraints
+    weight: 241
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
```
Comment (premkumr) [inline docs/content/preview/explore/indexes-constraints/multi-column-ycql.md:16]: Could you use `{{<<api-tabs>}}`

File: docs/content/preview/explore/indexes-constraints/multi-column-ysql.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    </a>
+  </li>
+
+  <li >
+    <a href="../multi-column-ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
```
Comment (premkumr) [inline docs/content/preview/explore/indexes-constraints/multi-column-ysql.md:30]: Could you use `{{<<api-tabs>}}`

**Review by premkumr — COMMENTED** [review]
LGTM

File: docs/content/preview/explore/indexes-constraints/multi-column-ysql.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/explore/indexes-constraints/multi-column-ysql.md]: yes! Thanks .

**Comment by aravind-nallan-yb** [comment]
The page on Secondary Indexes already says the column-list can include multiple columns separated by commas. This just seems like a special case of secondary indexes and not a different type index entirely like Unique indexes or Partial indexes. I'd just add examples in the Secondary Indexes page itself.

Also "These indexes provide faster access to data entries as it uses multiple columns to sort through data faster." I'm not sure this is true. Faster access compared to what? Single column indexes? Multiple columns are used to support multiple access patterns. Not meant for speed.

**Comment by aishwarya24** [comment]
> The page on Secondary Indexes already says the column-list can include multiple columns separated by commas. This just seems like a special case of secondary indexes and not a different type index entirely like Unique indexes or Partial indexes. I'd just add examples in the Secondary Indexes page itself.
> 
@yogesh-yb - Do we still need to have a separate page for multi-column index ?

**Comment by ddhodge** [comment]
I agree - it is an expanded case of secondary index, and secondary index page refers to this as compound index. Add this to the Secondary index page

File: docs/content/stable/explore/indexes-constraints/secondary-indexes-ycql.md [hunk 3]
```diff
@@ -132,6 +132,114 @@ To remove the index `index_employees_department`, use the following command:
...
+```output
+ QUERY PLAN
+------------------------------------------------------------
+ Seq Scan on example.employees
+   Filter: (first_name = 'Dave') AND (last_name = 'Marley')
+```
+
+To process the preceding query, the whole `employees` table needs to be scanned and all the shards have to be accessed. For large organizations, this will take a significant amount of time as the query is executed sequentially as demonstrated in the preceding EXPLAIN statement.
+
+To speed up the process, you can create an index for the department column, as follows:
```
Comment (ddhodge) [inline docs/content/stable/explore/indexes-constraints/secondary-indexes-ycql.md:223]: example creates index on first and last name, not department

File: docs/content/stable/explore/indexes-constraints/secondary-indexes-ycql.md [hunk 4]
```diff
@@ -132,6 +132,114 @@ To remove the index `index_employees_department`, use the following command:
 DROP INDEX index_employees_department;
 ```
 
+## Example scenario using multi-column index
+
+Multi-column indexes are also known as compound indexes using which you can create an index with multiple columns.
```
Comment (ddhodge) [inline docs/content/stable/explore/indexes-constraints/secondary-indexes-ycql.md:137]:
```suggestion
You can speed up queries that search on more than one column by creating a multi-column index.
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-ycql.md [hunk 5]
```diff
@@ -132,6 +132,114 @@ To remove the index `index_employees_department`, use the following command:
 DROP INDEX index_employees_department;
 ```
 
+## Example scenario using multi-column index
+
+Multi-column indexes are also known as compound indexes using which you can create an index with multiple columns.
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-ycql.md:137]: see ysql comment

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**